sass/sass.bzl

```diff
@@ -94,6 +94,8 @@ def _run_sass(ctx, input, css_output, map_output = None):
     # Note that the sourcemap is implicitly written to a path the same as the
     # css with the added .map extension.
     args.add_all([input.path, css_output.path])
+    args.use_param_file("@%s", use_always = True)
```

Comment:
Will this break in Google3, which will still invoke the Sass executable directly and thus won't understand a param file? If so, we should adjust copybara to remove these lines.

Reply:
Yes, we need to adjust copybara.

My understanding is that will be a google3 changelist and should not affect this PR. Do you have any pointers?

Reply:
I have changed the copybara to filter out these lines in google3.